src/controllers/login.js

```diff
@@ -0,0 +1,44 @@
+import UserService from '../services/UserService';
+import { isPasswordTrue } from '../utils/paaswordDecrypt';
+import generateToken from '../utils/tokenHandler';
+
+/**
+   * function findAll() returns all users in db
+   * @param {object} req it request
+   * @param {object} res it response
+   * @returns {object} send reponse to the user
+   */
+const getUser = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    const gottenUser = await UserService.findUserByEmail(email);
+
+    if (gottenUser) {
+      const { dataValues } = gottenUser;
+      if (await isPasswordTrue(password, dataValues.password)) {
+        const userInfo = {
+          names: dataValues.names,
+          email: dataValues.email
+        };
+        const token = await generateToken(userInfo);
+        res.status(200).json({
```

Comment:
could you create a helper function to manage the responses? you can refer to @the22mastermind's PR

Reply:
done, and thank you so much, this helped my function to be smaller than before